### DISCUSSION TITLE
xmake和cmake并存

### DISCUSSION THREAD

**Discussion by holdxen** [xmake-io/xmake-vscode#264] [Q&A]
vscode有没有办法：当项目中同时存在xmake和cmake时，暂时禁用cmake的插件，不让cmake的插件生成乱七八糟的东西，优先使用xmake。

**Comment by waruqi** [comment]
目前 cmake 和 xmake 的 toolbar 都在底下，同时开启，显示不下，我记得可以同时存在，但是同时激活用起来也乱。

只能自己配置禁用来管理。
